app/domain/authentication/authn_jwt/fetch_token_from_credentials.rb

```diff
@@ -0,0 +1,22 @@
+module Authentication
+  module AuthnJwt
+
+    FetchTokenFromCredentials ||= CommandClass.new(
+      dependencies: {
+        decoded_credentials_class: Authentication::Jwt::DecodedCredentials
+      },
+      inputs: %i[authentication_parameters]
+    ) do
+
+      def call
+        decoded_credentials(@authentication_parameters).jwt
+      end
+
+      private
+
+      def decoded_credentials(authentication_parameters)
+        @decoded_credentials_class.new(authentication_parameters.credentials)
+      end
```

Comment:
```suggestion
      def call
        decode_credentials
        extracted_token_from_credentials
      end

      private

      # this function is implemented so that the 'call' function will tell the story
      def decode_credentials
        decoded_credentials
      end

      def decoded_credentials
        @decoded_credentials ||= @decoded_credentials_class.new(@authentication_parameters.credentials)
      end

      def extracted_token_from_credentials
        decoded_credentials.jwt
      end
```